docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
     stdin_open: true
     working_dir: /home/package
     environment:
-      - MEILISEARCH_HOST=http://meilisearch:7700
+      - MEILISEARCH_URL=http://meilisearch:7700
```

Comment:
You should also change in every other place where we use the env var :)

Reply:
Like in the playground, spec/support/1_initialize_meilisearch.rb and other places :)